packages/vats/test/bootstrapTests/bench-vaults-performance.js

```diff
@@ -34,26 +34,34 @@ const snapshotHeap = async step => {
     const heapStats = v8.getHeapStatistics();
     const t3 = performance.now();
 
-    // process.pid increments so these will be lexically sorted pathnames.
-    const heapSnapshot = `Heap-${process.pid}-${snapshotNum}-${step}.heapsnapshot`;
-    snapshotNum += 1;
-
-    v8.writeHeapSnapshot(heapSnapshot);
-    const heapSnapshotTime = performance.now() - t3;
-
-    console.log(`HEAP DETAILS at step${step} vaults: `, {
+    const memStats = {
       memoryUsage,
       heapStats,
-      heapSnapshot,
       statsTime: {
         forcedGc: t1 - t0,
         memoryUsage: t2 - t1,
         heapStats: t3 - t2,
-        heapSnapshot: heapSnapshotTime,
       },
-    });
+    };
+
+    if (dumpHeap) {
+      console.log(`Snapshotting heap at step ${step}...`);
+
+      // process.pid increments so these will be lexically sorted pathnames.
+      const heapSnapshot = `Heap-${process.pid}-${snapshotNum}-${step}.heapsnapshot`;
+      snapshotNum += 1;
+
+      v8.writeHeapSnapshot(heapSnapshot);
+      const heapSnapshotTime = performance.now() - t3;
+      memStats.heapSnapshot = heapSnapshot;
+      memStats.statsTime.heapSnapshot = heapSnapshotTime;
+    }
+
+    console.log(`HEAP DETAILS at step${step} vaults: `, memStats);
```

Comment:
nit: Space needed between step${step}. 
Also `HEAP DETAILS` can be in sentence case similar to other logs.

Reply:
I just inherited that text, so...